app/views/roles/_fields.html.haml

```diff
@@ -0,0 +1,29 @@
+-# copied from core because I want to replace ALL the label-fields
+
+- if @group_selection.present? && @group_selection.size > 1
+  = f.labeled(:group_id) do
+    .span6.shown
+      = f.select(:group_id,
+                 group_options_with_level,
+                 { },
+                 { class: 'chosen-select',
+                   data: { remote: true,
+                           url: role_types_group_roles_path(@group),
+                           chosen_no_results: t('global.chosen_no_results') } })
+- else
+  = f.hidden_field :group_id
+
+= f.labeled(:type) do
+  #role_type_select.span6
+    = render 'roles/type_select'
+
+= field_set_tag Role.human_attribute_name(:label) do
+  - Settings.application.languages.each do |code, name|
+    = f.labeled_input_field(:"label_#{code}",
+                            data: { provide: :typeahead, source: entry.klass.available_labels },
```

Comment:
Müsste typeahead nicht auch noch in der entsprechenden Sprache sein? Oder vielleicht einfach in allen Sprachen? Oder noch einfacher gar kein typeahead?

Reply:
Guter Punkt. Ich passe es an, dass es die passende Sprache nimmt.